src/javascript/app/Stores/Modules/Contract/contract-store.js

```diff
@@ -56,8 +57,11 @@ export default class ContractStore extends BaseStore {
         const end_time                 = getEndTime(contract_info);
         const should_update_chart_type = (!contract_info.tick_count && !this.is_granularity_set);
 
-        SmartChartStore.setContractStart(date_start);
         if (end_time) {
+            if (this.is_from_positions) {
+                SmartChartStore.setStaticChart(true);
```

Comment:
@zakibinary this isn't working corectly when you open an ongoing contract from positions.
it will set static chart to true though it should be false.

Reply:
@MahboobehMohammadi, it's only setting `static` to `true` if the contract has ended and is opened from positions. Refer to the condition check for `end_time` on L60.

Reply:
@MahboobehMohammadi, anything else? If we're good, I'll mark this as resolved.